tests/unit/Services/ApmCollectorServiceTest.php

```diff
@@ -27,6 +27,8 @@ protected function setUp(): void
         $this->configMock = Mockery::mock(Config::class);
         $this->eventsMock = Mockery::mock(Dispatcher::class);
 
+        $this->appMock->shouldReceive('runningInConsole')
```

Comment:
With mockery you'll still need `->once()` or `expects()` (it's like `shouldReceive` but with implied once) to set the expectation, which I think you should in this case. E.g.:

```suggestion
        $this->appMock->expects('runningInConsole')
```